frontend/src/main/java/com/google/tripmeout/frontend/servlet/TripServlet.java

```diff
@@ -17,7 +17,7 @@
   private static final FluentLogger logger = FluentLogger.forEnclosingClass();
 
   private static final long serialVersionUID = 1L;
-  private static final String APPLICATION_JSON_CONTENT_TYPE = "application/json;";
+  private static final String APPLICATION_JSON_CONTENT_TYPE = "application/json";
```

Comment:
I am going to assume this was causing a problem, do you know why?

Reply:
based on what I read online, it seems like the flutter html package just doesn't like trailing semi-colons

Reply:
ok